OutOfSchool/OutOfSchool.WebApi/Services/ApplicationService.cs

```diff
@@ -943,21 +941,17 @@ private async Task<Result<ApplicationDto>> ExecuteUpdateAsync(ApplicationUpdate
 
                     if (amountOfApproved >= availableSeats)
                     {
-                        return Result<ApplicationDto>.Failed(new OperationError
-                        {
-                            Code = "400",
-                            Description = "The Workshop is full.",
-                        });
+                        badRequestApiErrorResponse.AddApiError(
```

Comment:
(Small code style suggestion) You could create and use small "helping" extension method here.
Reason: There are many places in code where `ApiErrorResponse` is created for the only `ApiError` and then immediately passed somewhere. Thus every time there is `var response = new ApiErrorResponse` and then `response.AddApiError(...)` and then `return ...response...`

You could create 
```
public static class ApiErrorExtensions {
    public static ApiErrorResponse ToResponse(this ApiError)
        => ...
}
```
And then use it as one-liner in your code. I.e., for the commented place it will look like:
```
return ErrorResponse.BadRequest(ApiErrorsTypes.Application.AcceptRejectedAlreadyApproved().ToResponse());
```